daliuge-translator/dlg/dropmake/pgtp.py

```diff
@@ -120,7 +120,10 @@ def to_partition_input(self, outf=None):
                 "self._drop_list, max RSS: %.2f GB",
                 resource.getrusage(resource.RUSAGE_SELF)[2] / 1024.0**2,
             )
-
+        tw = 1
+        sz = 1
+        dst = "outputs"
```

Comment:
I don't know if "outputs"/"inputs" are what we want to be defaults here, but it _is_ important to have defaults. An alternative may simply be empty strings.